cmd/neofs-lens/internal/fstree/cleanup.go

```diff
@@ -0,0 +1,30 @@
+package fstree
+
+import (
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	"github.com/spf13/cobra"
+)
+
+var cleanupCMD = &cobra.Command{
+	Use:   "cleanup-tmp",
+	Short: "Clean up tmp files in FSTree",
+	Long:  "Clean up temporary files in FSTree",
```

Comment:
Maybe some explanation here, it's not obvious what temporary files could be present there.